configure.ac

```diff
@@ -18,38 +18,16 @@ dnl (Make these variables available for use in Makefile.am)
 AM_CONDITIONAL([LINUX], [test -n "`echo ${target_os} | grep linux`"])
 AM_CONDITIONAL([MACOSX], [test -n "`echo ${target_os} | grep darwin`"])
 
-BOOST_VENDOR_DIR="`cd $(dirname $0); pwd`/vendor/boost"
-AC_CACHE_CHECK([whether Boost was bootstrapped (at ${BOOST_VENDOR_DIR})],
-               ax_cv_boost_bootstrapped,
-               AS_IF([test -f "${BOOST_VENDOR_DIR}/b2"],
-                     [ax_cv_boost_bootstrapped=yes],
-                     [ax_cv_boost_bootstrapped=no]
-                    )
-              )
-AS_IF([test "x$ax_cv_boost_bootstrapped" != "xyes"],
-      [AC_MSG_ERROR([Boost at ${BOOST_VENDOR_DIR} not boostrapped])]
-     )
-
-BOOST_INSTALL_DIR="${BOOST_VENDOR_DIR}/install"
-BOOST_CPPFLAGS="-I ${BOOST_INSTALL_DIR}/include"
-BOOST_LDFLAGS="-L${BOOST_INSTALL_DIR}/lib"
-AC_SUBST(BOOST_CPPFLAGS)
-AC_SUBST(BOOST_LDFLAGS)
-AC_MSG_NOTICE([BOOST_CPPFLAGS : ${BOOST_CPPFLAGS}])
-AC_MSG_NOTICE([BOOST_LDFLAGS : ${BOOST_LDFLAGS}])
-
-BOOST_JSON_LIB="-lboost_json"
-AC_SUBST(BOOST_JSON_LIB)
-AC_MSG_NOTICE([BOOST_JSON_LIB : ${BOOST_JSON_LIB}])
-
-AS_IF([test x"$host" != x"$build"],
-      [AC_MSG_NOTICE([Cross-compiling for ${host_cpu}])
-       echo "using gcc : ${host_cpu} : \"${CXX}\" ;" >> ${BOOST_VENDOR_DIR}/project-config.jam
-       BOOST_TOOLSET="toolset=gcc-${host_cpu}"
-      ],
-      [BOOST_TOOLSET=""]
-     )
-AC_SUBST(BOOST_TOOLSET)
+dnl TODO: Check which is the actual minimum we support!
+AX_BOOST_BASE([1.74.0],
+    [AC_SUBST([boost_CPPFLAGS], [${BOOST_CPPFLAGS}])
+        AC_MSG_NOTICE([boost_CPPFLAGS : ${boost_CPPFLAGS}])
+        AC_MSG_NOTICE([boost_LDFLAGS : ${boost_LDFLAGS}])],
+    [AC_MSG_ERROR([Boost 1.74.0 or later is required but was not found.])])
+
+AX_BOOST_JSON
+AC_SUBST([boost_json_LIBS], [${BOOST_JSON_LIB}])
+AC_MSG_NOTICE([boost_json_LIBS : ${boost_json_LIBS}])
```

Comment:
Revert (`reset -p`?) artifact here.

Reply:
Hmm, I'm not following. What do you mean?

Reply:
Oh, I thought you were removing it completely. But you are, **properly**, first just making it not vendored. :+1:

Reply:
Yes, I wanted to have every commit be functional still :smiley_cat: